src/studio/src/designer/frontend/ux-editor/components/toolbar/InformationPanelComponent.tsx

```diff
@@ -89,7 +88,7 @@ class InformationPanel extends React.Component<IInformationPanelProps> {
           }
         </Typography>
         <Typography classes={{ root: classNames(this.props.classes.informationPanelLink) }}>
-          <Link to={'/designsystem'}>{this.props.language.ux_editor.information_more_info_link}</Link>
+          <a href='https://docs.altinn.studio/teknologi/altinnstudio/solutions/altinn-studio/functional/build-app/ui-designer/components/'>{this.props.language.ux_editor.information_more_info_link}</a>
```

Comment:
I guess we should add `target='_blank'` to open in a new tab? As this is new an actual external link

Reply:
I did, but got a code smell cause using _blank wasn't secure.. Should we still add it?

Reply:
I guess you also have to add ` rel='noopener noreferrer'`.